app/modules/ads/index.js

```diff
@@ -413,6 +413,22 @@ class Ads {
     }
   }
 
+  /**
+   * @private
+   */
+  trackKruxSegments() {
+    const { context, utils } = window.Wikia.adEngine;
+    const kruxUserSegments = context.get('targeting.ksg') || [];
+    const kruxICBMSegments = context.get('services.krux.icbmSegments') || [];
```

Comment:
Can we rename it to `kruxTrackedSegments` and `services.krux.trackedSegments`? I don't see the point of using ICBM in the variable names since almost all context values comes from ICBM.